src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/PrisonerResourceTest.kt

```diff
@@ -15,16 +15,40 @@ class PrisonerResourceTest : ResourceTest() {
   @Nested
   @DisplayName("GET /api/prisoners")
   inner class GetPrisoners {
+
     @Test
-    fun testCanFindSinglePrisoner() {
-      val httpEntity = createEmptyHttpEntity(GLOBAL_SEARCH)
-      val response = testRestTemplate.exchange(
-        "/api/prisoners?offenderNo=A1234AC",
-        HttpMethod.GET,
-        httpEntity,
-        object : ParameterizedTypeReference<String?>() {},
-      )
-      assertThatJsonFileAndStatus(response, HttpStatus.OK, "prisoners_single.json")
+    fun `returns 401 without an auth token`() {
```

Comment:
accidentally added tests to wrong endpoint! but ok to leave in before change is made to this endpoint too